doc/classes/MultiMesh.xml

```diff
@@ -136,5 +158,11 @@
 		<constant name="CUSTOM_DATA_FLOAT" value="2" enum="CustomDataFormat">
 			The [Color] passed into [method set_instance_custom_data] will use 4 floats. Use this for highest precision.
 		</constant>
+		<constant name="INTERP_QUALITY_FAST" value="0" enum="PhysicsInterpolationQuality">
```

Comment:
I think it should be `INTERP_QUALITY_LOW` to be consistent with other quality enums in Godot. We use a `low/high` dichotomy as opposed to `fast/high`.

Reply:
I'm not absolutely sure this would give the correct impression. In the vast majority of cases fast _isn't_ low quality, it is just faster. If an object is not rotating at all, fast produces the exact same result as high quality, and unless an object is rotating at high angular velocity, the quality of fast is not significantly worse than high quality.

If we call something "low quality", people are more likely to assume that they need to use "high quality" for the best result, whereas this is not the case here. "fast" is purposefully vague on this, and the tooltip points out that "high quality" is advantageous only in some specific situations.